saskatoon/sitebase/templates/app/list_views/community/data_row.html

```diff
@@ -24,15 +24,17 @@
         </span><br>
         <a href='mailto:{{user.email}}'>{{ user.email }}</a> &nbsp;
         {% if perms.member.change_authuser %}
-            <a href="/admin/member/authuser/{{user.id}}/password/"
-               onclick="return confirm('{% trans "Are you sure you want to reset this user password?" %}');"
-            {% if not user.password %}
-                title="register-user"
-                >
-                <small class="btn btn-primary  waves-effect btn-sm"><i class="fa fa-plus"> </i> {% translate ' Register User' %} </a></small>
-            {% else %}
-                title="reset-password">
-                <small><i class="fa fa-refresh"></i></a></small>
+            {% if 'core' in user.role_codes or 'pickleader' in user.role_codes %}
```

Comment:
I think only core members or admins should be able to reset one's password. Not pickleaders.

Reply:
Ok sorry for my confusion.